tests/test_cache_embeddings.py

```
@@ -322,3 +322,24 @@ async def test_in_memory_cache_stores_multiple_texts():
+
+    Regression test: EmbeddingsCache.from_dict built the cache store from
+    store_config but never forwarded store_config to the constructor, so
+    get_config() reported an empty store_config and the configured store
+    location (e.g. a custom filesystem cache_dir) was silently dropped on a
+    config round-trip.
+    """
+    cfg = EmbeddingsCacheConfig(
+        enabled=True,
+        key_generator="md5",
+        store="filesystem",
+        store_config={"cache_dir": "/tmp/myembcache"},
```

Ruff (0.15.14)

[error] Probable insecure usage of temporary file or directory: "/tmp/myembcache"

(S108)